src/DbStorageImpl.cc

```diff
@@ -50,6 +50,7 @@ static char const* SVNid __attribute__((unused)) = "$Id$";
 #include <stdlib.h>
 #include <unistd.h>
 #include <vector>
+#include <memory>
```

Comment:
Should be alphabetically sorted ([coding standard 4-10](http://developer.lsst.io/en/latest/coding/cpp_style_guide.html#include-statements-should-be-sorted-and-grouped)).